lib/index.ts

```diff
@@ -1,136 +1,141 @@
 
             if (!req.query.peer_id ||
                 !req.query.to) {
-                return res.status(400).end()
+                return res.status(400).end();
             }
 
             // find the current peer
-            const peer = router.peerList.getPeer(req.query.to)
+            const peer = router.peerList.getPeer(req.query.to);
 
             if (!peer) {
-                return res.status(404).end()
+                return res.status(404).end();
             }
 
             // send data to the peer
             // (this will write to the `to` socket, or buffer if needed)
-            sendPeerMessage(req.query.peer_id, req.query.to, req.body)
+            sendPeerMessage(req.query.peer_id, req.query.to, req.body);
 
             // whether we send directly or buffer we tell the sender everything is 'OK'
-            res.status(200).end()
-        }
-    )
+            res.status(200).end();
+        },
+    );
 
-    router.get('/wait', (req, res) => {
+    router.get("/wait", (req, res) => {
         if (!req.query.peer_id) {
-            return res.status(400).end()
+            return res.status(400).end();
         }
 
-        const pop = router.peerList.popPeerData(req.query.peer_id)
+        const pop = router.peerList.popPeerData(req.query.peer_id);
 
         // if we have data to send, just send it now
         if (pop) {
             return res.status(200)
-                .set('Pragma', pop.srcId)
-                .send(pop.data)
-        }
-        // otherwise, capture the socket so we can write to it later
-        else {
+                .set("Pragma", pop.srcId.toString())
+                .send(pop.data);
+        } else {
             // set the socket for the given peer and let it hang
             // this is the critical piece that let's us send data
             // using 'push'-ish technology
-            router.peerList.setPeerSocket(req.query.peer_id, res, req)
+            router.peerList.setPeerSocket(req.query.peer_id, res, req);
         }
-    })
+    });
 
-    router.get('/sign_out', (req, res) => {
+    router.get("/sign_out", (req, res) => {
         if (!req.query.peer_id) {
-            return res.status(400).end()
+            return res.status(400).end();
         }
 
         // remove the peer
-        router.peerList.removePeer(req.query.peer_id)
+        router.peerList.removePeer(req.query.peer_id);
 
         // send an updated peer list to all peers
-        router.peerList.getPeerIds().forEach((id) => {
+        router.peerList.getPeerIds().forEach((id: number) => {
             // updated peer lists must always appear to come from
             // "ourselves", namely the srcId == destId
-            sendPeerMessage(id, id, router.peerList.dataFor(id))
-        })
+            sendPeerMessage(id, id, router.peerList.dataFor(id));
+        });
+
+        res.status(200).end();
+    });
+
+    return router;
+}
 
-        res.status(200).end()
-    })
+export {
```

Comment:
why export all these here rather than having the consumer call-sites reference `./modules` directly to import them (with the exception of `signalRouterCreator` - i agree that one should be there).

Reply:
Not sure what you mean. Do you mean using 'import webrtc-signal-http/modules'?

Reply:
ah are consumers outside of this repo? if so this pattern makes sense.

I was thinking you'd done this so that elsewhere in this repo you could `import { ThingFromModules } from './'` instead of ` import { ThingFromModules } from './modules'`

Reply:
yep! there are consumers in 3d-signal